content/en/blog/preact-x.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Preact X, a story of stability
+date: 2024-05-24
+authors:
+  - Jovi De Croock
+---
+
+# Preact X, a story of stability
+
+A lot of you have been waiting for [Preact 11](https://github.com/preactjs/preact/issues/2621), announced in an issue opened
+way back in July 2020, and to be clear I was one of the most excited people for v11.
+When we started thinking about Preact 11 we believed that there was no way to introduce the changes we had in mind
+in Preact X without breaking changes, some of the things we had in mind:
+
+- Using a backing VNode structure to reduce GC, by doing this we'd only use the result of `h()` to update our backing-node.
+- Reconciler performance, adding fast paths for mounting/...
+- Some changes like removing `px` suffixing, `forwardRef` and breaking IE11 support.
+- Addressing event/child diffing bugs.
```

Comment:
Should we add the `keeping ref in props` change here. We had to backtrack to react compat.